hivemind/optim/experimental/optimizer.py

```diff
@@ -461,7 +461,7 @@ def _update_global_epoch(self, grad_scaler: Optional[GradScaler]) -> None:
                 if not self.client_mode:
                     self.grad_averager.state_sharing_priority = self.local_epoch
 
-            logger.log(self.status_loglevel, f"Transitioning to epoch {self.local_epoch}.")
+            logger.debug(f"Transitioning to epoch {self.local_epoch}")
```

Comment:
This duplicates the `Step N` message (see the "Above" screenshot).

Reply:
This message is useful because
- it signifies when a (potentially long) optimizer step is finished
- it is the only message that will be printed when use_local_updates=True (i.e. there is no global step)